plantseg/viewer/widget/proofreading/proofreading.py

```diff
@@ -248,6 +248,11 @@ def widget_split_and_merge_from_scribbles(viewer: napari.Viewer,
     if image is None:
         napari_formatted_logging('Image Layer not defined', thread='Proofreading tool', level='error')
         return None
+    elif 'pmap' not in image.metadata:
+        napari_formatted_logging('Pmap/Image layer appears to be a raw image and not a pmap. For the best proofreading '
```

Comment:
minor: it's fine to keep the `Pmap/Image` on the button, but I'd avoid the abbreviation `pmap` in the warning and just say `probability maps` to make it clear.